pulp_container/app/authorization.py

```diff
@@ -203,20 +205,17 @@ def has_push_permissions(self, path):
 
     def has_view_catalog_permissions(self, path):
```

Comment:
```suggestion
    def has_view_catalog_permissions(self):
```
Would this work too? The catalog has no `path`.

Reply:
All the methods which check for permissions have the same input parameters and are called in the loop: https://github.com/pulp/pulp_container/blob/0de33d42bf5f86b11d29a99757b5f6a53cca8d40/pulp_container/app/authorization.py#L148

Furthermore, we do not check anywhere whether the path is truly "catalog" or not. That is why I left the path parameter there.

Reply:
Isn't the path `_catalog` then?

https://docs.docker.com/registry/spec/api/#listing-repositories

Reply:
No, it is not. Because we are parsing the scope string (scope="registry:catalog:*") and not the endpoint itself during the authorization.